### PR TITLE
Testing | Packaging v2

### PR DESCRIPTION
**Pull request by alexAubin**

### CODE REVIEW

**Review by kay0u — APPROVED** [review]
LGTM

**Comment by kay0u** [comment]
!testme

File: scripts/_common.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    fi
+
+    # Avoid uncrypted remote destination with reverse proxy mode
+    # Indeed the SSO send the password in all requests in HTTP headers
+    if [[ "$redirect_type" = "reverseproxy" ]] && [[ ! $target =~ $URL_REGEX_SECURE ]]; then
+        ynh_die --message="For secure reason, you can't use an unencrypted http remote destination couple with ssowat for your reverse proxy: $target" 1
+    fi
+}
+
+_add_dummy_service() {
```
Comment (Salamandar) [inline scripts/_common.sh]: IMHO it would be best to install another app (my_webapp) in the preinstall in tests.toml.

**Review by Salamandar — APPROVED** [review]

**Comment by alexAubin** [comment]
!testme

File: scripts/_common.sh (hunk 1 above)
Comment (alexAubin) [reply scripts/_common.sh]: Hmmyeah but my_webapp ain't really a process listening to a specific port so eh .. but yeah we could try to find some super-simple app that does run a systemd service

**Pull request merged**